stm32-modules/flex-stacker/firmware/system/i2c_hardware.c

```diff
@@ -1,5 +1,9 @@
 #include <stdint.h>
 
+#pragma GCC push_options
+#pragma GCC optimize("O0")
```

Comment:
we sure about this one? What problem is it solving?

Reply:
nope, this was for debugging, will remove it.